client/state/plugins/installed/test/selectors-ts.ts

```diff
@@ -310,6 +311,18 @@ describe( 'getFilteredAndSortedPlugins', () => {
 	} );
 } );
 
+describe( 'getPluginsWithUpdates', () => {
+	test( 'Should get an empty array if the requested site is not in the current state', () => {
+		const plugins = getPluginsWithUpdates( state, [ nonExistingSiteId1 ] );
+		expect( plugins ).toHaveLength( 0 );
+	} );
+
+	test( 'Should get a plugin list of length 1 when we can update files on the site', () => {
+		const plugins = getPluginsWithUpdates( state, [ siteOneId, siteTwoId ] );
+		expect( plugins ).toHaveLength( 1 );
```

Comment:
It may be more precise to verify that the plugin in the list is the expected one.